src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -85,6 +87,20 @@ public void addMessage(Message message) {
     return messagesInConversation;
   }
 
+  /** Access the set of Messages sent by the user. */
+  public List<Message> getMessagesByUser(UUID author) {
+
+    List<Message> messagesByUser = new ArrayList<>();
```

Comment:
This variable name seems to suggest a map or hash table, I'd just say userMessages instead